src/components/nc-list.vue

```diff
@@ -0,0 +1,71 @@
+<template>
```

Comment:
En cuanto a estilos

- ¿Dejamos por ahora el tema del check para mostrar que se ha seleccionado un item concreto?
- Deberíamos quizás alinear a la izquierda el span con el país y a la derecha el span con el código del país
- ¿propiedad para poner color a los span? (uno más oscuro otro más claro)

Todo ello según lo que negociemos con Miriam, pero al menos quizás separar los contenidos de los span e indicar qué item se ha seleccionado

Reply:
Tienes razón. La cuestión es llevar esa responsabilidad a un componente nc-list-item que tendríamos que crear. El nc-list haría un v-for de ese componente, pero es el nc-list el que tiene que recibir los items para filtrar la lista y vincular la lista con el v-model=search.
Esta petición la trasladaría a ese componente nc-list-item, para el que generamos los diferentes tipos de item de lista que nos pida diseño.

De todos modos, podemos planificar la creación del nc-list-item para el próximo sprint y por ahora usar un input para que el usuario meta el código de país a mano. Piensa que tenemos también que definir un servicio para pasarle todos los códigos de país al componente.